gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -74,7 +75,10 @@
 _LARGE_DATA_THRESHOLD = 50000
 _MERGE_HEADERS_FILE_NAME = 'merged_headers.vcf'
 _MERGE_HEADERS_JOB_NAME = 'merge-vcf-headers'
-
+# The first 22 partitions [0, 21] are reserved for standard reference_name
+# formated as RegExp '^(c|ch|chr)?([0-9][0-9]?)$'.
+# Every other identifers will be matched to next available partitions [22, 26].
+_DEFAULT_PARTITION_NO = 22 + 5
```

Comment:
I see. Thanks for the extra context. However, I have been thinking about this and I'm a bit uneasy about having this random number in our main module, and later the PartitionConfig class added here as well. I think a better approach could be:
- Create a new libs.variant_partition.VariantPartition class that inherits from object (i.e. simple lib class).
- Define this constant in that module.
- Implement a `get_num_partitions()` method that, for now, just returns that constant. Later on, it can parse the config file and return the appropriate value.
- Implement a `partition(variant)` method inside that lib and move all of the partitioning logic there.
- Change PartitionVariants to get the VariantPartition lib as an arg. The partition_for method simply does `return variant_partition.partition(variant)`. You may actually choose to remove the ParitionVariants class altogether since it's just a dummy class now and can call VariantPartition.partition directly (you'd have a dummy num_partitions arg, but that's ok).

Finally, the code in main becomes something like:
```
variant_partition_lib = VariantPartition() # this can later take in config_file_path
partitions = variants | 'PartitionVariants' >> beam.Partition(variant_parition_lib.partition, variant_parition_lib.get_num_partitions())
merged = [... for i in xrange(variant_parition_lib.get_num_partitions())]
...
```

The benefit of this approach is that all of the partitioning logic lives in one place and we can add/remove logic about handling config files without affecting the main module.
WDYT? I'm very open to other ideas as well as long as it removes the constant from main :)